test-lib.sh

```diff
@@ -322,40 +349,54 @@ EOF
 # Uses: $IMAGE_NAME - name of the image being tested
 function ct_check_exec_env_vars() {
   local tmpdir exec_envs cid old_IFS env_filter
-  local var_name stripped filtered_envs
+  local var_name stripped filtered_envs loop_envs
   env_filter=${1:-"^X_SCLS=\|/opt/rh\|/opt/app-root"}
   tmpdir=$(mktemp -d)
   CID_FILE_DIR=${CID_FILE_DIR:-$(mktemp -d)}
   # Get environment variables from `docker run`
-  docker run --rm "$IMAGE_NAME" /bin/bash -c "env" | sort > "$tmpdir/run_envs"
+  loop_envs=$(docker run --rm "$IMAGE_NAME" /bin/bash -c "env")
```

Comment:
Why `loop_envs`? `run_envs` was better

Reply:
I wonder how that went in here. Probably some copy paste error while moving the code. Will change back. Thanks for spotting this!